low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -55,23 +55,19 @@ SSPL_LL_SETTING:
       - Smartctl
```

Comment:
core_processors and message_handlers i dont think should be configurable and can remain in sspl_constants. Core processors are must for all functionalities and message handles based on sensors & actuators enabled, which will complicate matters if we enable/disable handler run time

Reply:
Moved processors and msg_hanlders in sspl_constants file.